### DISCUSSION TITLE
HEY

### DISCUSSION THREAD

**Discussion by 12Minhnhut34** [community/community#137343] [Actions]
### Select Topic Area

Question

### Body

HEY DO YOU SEE MY PROJECT IT EASY

**Comment by SaumyaSubham** [comment]
Hey 12Minhnhut34, What's up? And Yes, I saw it!

**Comment by AkshayMoolya** [comment]
The only project I can find in your repository is a Java console application, which is currently empty.

**Comment by samus-aran** [comment]
Hi @12Minhnhut34 , we’re glad you’re here 😀

Thanks for sharing your project (?) with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
